src/player.cpp

```diff
@@ -1944,6 +1945,9 @@ void player::process_items()
     if( ch_UPS_used > 0 ) {
         use_charges( "UPS", ch_UPS_used );
     }
+    if( ch_AIR_used > 0 ) {
```

Comment:
It believes that this condition will never be true.

Reply:
`ch_AIR_used` is initialized to zero and never changed, so this comparison can never be true.